web-app/js/portal/ui/openlayers/layer/NcWMS.js

```diff
@@ -160,9 +160,7 @@ OpenLayers.Layer.NcWMS = OpenLayers.Class(OpenLayers.Layer.WMS, {
 
     loadTimeSeriesForDay: function(date) {
         if (this.getTimeSeriesForDay(date)) {
-            // Give UI precedence by using delayed function call (setTimeout)
-            var that = this;
-            setTimeout(function() { that._timeSeriesLoadedForDate(); }, 10);
+            this._timeSeriesLoadedForDate();
```

Comment:
This change looks unrelated, to me. Have you rebased this branch off master recently?